var/spack/repos/builtin/packages/er/package.py

```diff
@@ -29,6 +31,7 @@ def cmake_args(self):
         if self.spec.satisfies('platform=cray'):
             args.append("-DER_LINK_STATIC=ON")
         args.append("-DWITH_KVTREE_PREFIX=%s" % self.spec['kvtree'].prefix)
+        args.append("-DWITH_RANKSTR_PREFIX=%s" % self.spec['rankstr'].prefix)
```

Comment:
I think this line will break for earlier versions, since `rankstr` is not a dependency in that case.

Reply:
Ah, yup. Stuck this in a conditional.